src/libraries/Common/tests/TestUtilities/System/PlatformDetection.Windows.cs

```diff
@@ -248,31 +248,8 @@ public static bool IsInAppContainer
             }
         }
 
-        public static bool CanRunImpersonatedTests => PlatformDetection.IsNotWindowsNanoServer && PlatformDetection.IsWindowsAndElevated;
+        public static bool CanRunImpersonatedTests => PlatformDetection.IsNotWindowsNanoServer && PlatformDetection.IsWindows && PlatformDetection.IsPrivilegedProcess;
```

Comment:
Will PlatformDetection.IsWindows ever be false here?

Reply:
Yes. Despite the `.Windows.cs`, file name, the file is unconditionally compiled so Windows-specific helpers like this are defined similarly throughout this file.